apps/prairielearn/elements/pl-threejs/pl-threejs.py

```diff
@@ -494,17 +494,17 @@ def grade(element_html, data):
 def parse_correct_answer(f, a):
     if f == "homogeneous":
         try:
-            T = np.array(a, dtype=np.float64)
-            if T.shape == (4, 4):
-                R = T[0:3, 0:3]
-                p = T[0:3, 3:4]
-                return np.reshape(p, (3,)), pyquaternion.Quaternion(matrix=R)
+            rotation_matrix = np.array(a, dtype=np.float64)
+            if rotation_matrix.shape == (4, 4):
+                rot_matrix = rotation_matrix[0:3, 0:3]
+                p = rotation_matrix[0:3, 3:4]
+                return np.reshape(p, (3,)), pyquaternion.Quaternion(matrix=rot_matrix)
```

Comment:
Uhhh, is this right? in a previous iteration `T` was renamed to `transform_matrix`, why is that now `rotation_matrix`, and why do we have both `rotation_matrix` and `rot_matrix`?

Reply:
Oops!

Reply:
Fixed in 42acf8ffc.